app/controllers/contacts_controller.rb

```diff
@@ -35,15 +35,17 @@ def get_contact_json
     # our DB to become an email -> instagram/name lookup for people to abuse
     ret = @contact.as_json.slice('id')
     ret[:instagram] = @contact.instagram.present?
+    ret[:is_eligible_for_lyft_reward] = @contact.is_eligible_for_lyft_reward
```

Comment:
any thoughts on consolidating these two fields? Like just have `is_eligible_for_lyft_reward` be false if `has_redeemed_lyft_reward` is true

Reply:
Yeah, we could definitely do that. Does the UI need `has_redeemed_lyft_reward` at all (do we want to show some badge or some status component)? If not, I can merge these.

Reply:
I agree that this might be better handled on the backend. This was outlined as an alt approach here: https://docs.google.com/document/d/1t4J91S21iFQkPrDa9IkUIWcYxoASjZS7B-d3BEwFgfc/edit#heading=h.lk2v4pj3i8vi

Reply:
It currently checks that value first here: https://github.com/sendchinatownlove/ruby/pull/240/files#diff-c4b0c8c9c2b6974a9038fdb3f22c11c4R27. I'm going to hide the flag from the response for now - we can expose it later if needed.